eclair-node/pom.xml

```diff
@@ -49,25 +49,24 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>com.github.chrisdchristo</groupId>
-                <artifactId>capsule-maven-plugin</artifactId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.2.0</version>
+                <configuration>
+                    <descriptors>
+                        <descriptor>modules/eclair-node.xml</descriptor>
+                    </descriptors>
+                    <archive>
+                        <manifest>
+                            <mainClass>fr.acinq.eclair.Boot</mainClass>
+                        </manifest>
+                    </archive>
+                </configuration>
                 <executions>
                     <execution>
+                        <phase>package</phase>
                         <goals>
-                            <goal>build</goal>
+                            <goal>single</goal>
                         </goals>
-                        <configuration>
-                            <appClass>fr.acinq.eclair.Boot</appClass>
-                            <type>fat</type>
-                            <fileName>${project.name}-${project.version}</fileName>
-                            <fileDesc>-${git.commit.id.abbrev}</fileDesc>
-                            <manifest>
-                                <entry>
-                                    <key>Java-Agents</key>
-                                    <value>kanela-agent-1.0.1.jar</value>
```

Comment:
How will agents be dealt with?

Reply:
Good catch i have to re-include it in the jar manifest, note that the [launcher](https://github.com/ACINQ/eclair/pull/1307/files#diff-282a1410413f84c78de1260977c66120R162) has support for java agents (although it needs to be updated)

Reply:
At 106224c the kanela agent is disabled by default but can be activated using the `-with-kanela` cmd line option when launching eclair-node or eclair-node-gui.